### PR TITLE
feat: Times of India extractor

### PR DESCRIPTION
**Pull request by nitinthewiz**
Adding a custom parser for the [Times of India](https://timesofindia.indiatimes.com). It's a messy site, no dek (hence no test for it), the author and publish dates are a single line of text, so it's difficult to parse, hence using the fallback for the GenericAuthorExtractor.

Tests pass just fine - 

```
Test Suites: 221 passed, 221 total
Tests:       1205 passed, 1205 total
Snapshots:   0 total
Time:        31.388s, estimated 32s
Ran all test suites.
```

My first PR, and hopefully more to come. I'd love feedback on this PR!

### CODE REVIEW

File: src/extractors/custom/timesofindia.indiatimes.com/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  domain: 'timesofindia.indiatimes.com',
+
+  title: {
+    selectors: ['h1'],
+  },
+
+  extend: {
+    reporter: {
+      selectors: ['div.byline'],
+      transforms: {},
```
Comment (adampash) [inline src/extractors/custom/timesofindia.indiatimes.com/index.js]: You can exclude the `transforms` here if you're not using it.

File: src/extractors/custom/timesofindia.indiatimes.com/index.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        $node.empty();
+        return $node;
+      },
+      '.byline': function remove_byline($node) {
+        $node.empty();
+        return $node;
+      },
+      '.img_cptn': function remove_img_cptn($node) {
+        $node.empty();
+        return $node;
```
Comment (adampash) [inline src/extractors/custom/timesofindia.indiatimes.com/index.js]: It seems like what you want to use instead here is the `clean` property. See [here](https://github.com/postlight/mercury-parser/blob/master/src/extractors/custom/README.md#cleaning-content-from-an-article).

Comment (nitinthewiz) [reply src/extractors/custom/timesofindia.indiatimes.com/index.js]: Oh shucks, yeah, should have remembered to use clean! I'll update the code and send over an updated PR for your consideration 😄

File: src/extractors/custom/timesofindia.indiatimes.com/index.js (hunk 1 above)
Comment (nitinthewiz) [reply src/extractors/custom/timesofindia.indiatimes.com/index.js]: done! :)

**Comment by nitinthewiz** [comment]
PASS  src/extractors/custom/timesofindia.indiatimes.com/index.test.js (9.539s)
  TimesofindiaIndiatimesComExtractor
    initial test case
      ✓ is selected properly (1ms)
      ✓ returns the title
      ✓ returns the author
      ✓ returns the date_published
      ✓ returns the lead_image_url (1ms)
      ✓ returns the content (1ms)

Test Suites: 1 passed, 1 total
Tests:       6 passed, 6 total
Snapshots:   0 total
Time:        10.354s
Ran all test suites matching /timesofindia.indiatimes.com/i.

**Comment by nitinthewiz** [comment]
Bump @adampash

**Review by JadTermsani — APPROVED** [review]

**Comment by JadTermsani** [comment]
@nitinthewiz included in the [latest release](https://github.com/postlight/mercury-parser/releases/tag/v2.2.1)!

**Pull request merged**